src/motion-transition/motion-transition.c

```diff
@@ -364,14 +372,15 @@ OBS_DECLARE_MODULE()
 OBS_MODULE_USE_DEFAULT_LOCALE("motion-transition", "en-US")
 
 struct obs_source_info motion_transition = {
-	.id = "motion-transition",
+	.id = "motion_transition",
```

Comment:
Why did we change the id? This will cause OBS import not being able to import correct.

Reply:
all other our inputs and transitions have underscore in names. changed to match. 
also in js code they cannot use - in variable names.

Reply:
Got it, let's find a solution next week for the obs importer, approving this solution for now.